extensions/azurecore/src/account-provider/auths/azureAuth.ts

```diff
@@ -0,0 +1,491 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
This is the landing area for the azure auth experience. It holds common code between device code and code grant methods.